Oqtane.Client/Modules/Controls/LocalizableComponent.cs

```diff
@@ -18,7 +18,7 @@ protected string Localize(string name)
         {
             if (!IsLocalizable)
             {
-                return null;
+                return name;
```

Comment:
Why?

Reply:
For modules that have already been created for Oqtane 1.x they may have used the ActionDialog control. When trying to run these modules on Oqtane 2.0 (.NET5) the modules run fine however when the ActionDialog modal popup is displayed, the buttons do not have any text. I stepped through the code and it is because the Localize() method is returning null. This appears to be  a breaking change as it affects backward compatibility. I am not sure if the solution I checked in is valid - if not, then we will need some other type of solution to deal with this issue. It does seem to make sense to me that if a component is not localizable then the method should the text which was passed in?

Reply:
Aha, I did that because you mentioned this in the PR about hating default values in localization ;)